src/pages/Folder/index.jsx

```diff
@@ -0,0 +1,88 @@
+import Footer from "components/Footer";
+import CardList from "components/CardList";
+import FolderHeader from "./components/FolderHeader";
+import FolderAddLinkArea from "./components/FolderAddLinkArea";
+import FolderCategory from "./components/FolderCategory";
+import FolderControl from "./components/FolderControl";
+import styles from "./styles.module.css";
+import SearchIcon from "assets/Search.png";
+import { useEffect, useState } from "react";
+import { getFolderList, getLinks } from "api/api";
+
+export default function Folder() {
+  const [search, setSearch] = useState("");
+  const [folders, setFolders] = useState([]);
+  const [selectedName, setSelectedName] = useState("전체");
+  const [selectedId, setSelectedId] = useState(null);
```

Comment:
P3:
제 취향에는 이 state를 하나로 관리해도 좋을 것 같아요. 따로 관리해야할 이유가 없어보이네요.

```suggestion
  interface FolderProps {
    name?: string;
    id?: number;
  }
  ...
  const [selectedFolder, setSelectedFolder] = useState<FolderProps>();
```